### ISSUE TITLE
Pre-Processing Module

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/oceanbench#2]
**TLDR**

This module will aggregate all of the data sources and apply some minimal consistent preprocessing strategies for relevant datasets. It will save the "cleaned" data for later preprocessing.

---
**Demo Steps**

1. Load Data from bucket, e.g. along track, gridded
2. Clean Labels, e.g. ssh, sst, u, v
3. Clean Coordinates, e.g. lat, lon, time
4. Save data

---
**Must Haves**:

- [x] Coordinate Correction (lat,lon) - [Example](https://github.com/jejjohnson/ml4ssh/blob/main/inr4ssh/_src/preprocess/spatial.py)
- [ ] Add units - [Example](https://pint-xarray.readthedocs.io/en/latest/generated/xarray.Dataset.pint.quantify.html)
- [x] Rename coordinate variables 
  - [x] Full names - `latitude`, `longitude`, `depth`, `swath`
  - [x] Short names - `lat`, `lon`, `z`, `w`
- [x] Rename variables
  - [x] Full names - `sea surface height`, `sea surface temperature`, ...
  - [x] Short names - `ssh`, `sla`, `mdt`, `sst`, `sss`, `ssc`,
